auth0/data_source_auth0_client_test.go

```diff
@@ -0,0 +1,149 @@
+package auth0
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/alexkappa/terraform-provider-auth0/auth0/internal/random"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+const testAccDataClientConfigInit = `
```

Comment:
This seems to be exactly the same as `testAccClientConfig in resource_auth0_client_test.go`. We could just reuse that to instantiate the resource so we can read the data.